### PR TITLE
Address page

### PR DESCRIPTION
**Pull request by Priya-Patel31**
Preview Link : https://deploy-preview-9--hopsop.netlify.app/address_page/address_page.html

### CODE REVIEW

File: address_page/address_page.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    .address-info {
+      font-size: 1.1rem;
+      margin-bottom: 0.2rem;
+    }
+    @media only screen and (max-width:550px){
+        .addresses-main-container{
+            margin:2rem 0.5rem;
+        }
+    }
+
```
Comment (Ankur9669) [inline address_page/address_page.css]: Suugestion: Indentation can be improved

**Review by Ankur9669 — APPROVED** [review]
The UI looks awesome!!!. Looks good to merge

**Review by arjundevn — APPROVED** [review]
Good to merge

**Comment by Priya-Patel31** [comment]
> _ No description provided. _

Preview Link : https://deploy-preview-9--hopsop.netlify.app/address_page/address_page.html

**Review by t007rushi — APPROVED** [review]
Add some description and link to direct page and not home page, UI looks good

File: address_page/address_page.css (hunk 1 above)
Comment (Priya-Patel31) [reply address_page/address_page.css]: done !!

**Pull request merged**